onnx/defs/quantization/defs.cc

```diff
@@ -101,4 +102,49 @@ ONNX_OPERATOR_SET_SCHEMA(
               updateOutputShape(ctx, 0, input_shape);
             }));
 
+static const char* FusedQuantizeLinear_ver11_doc = R"DOC(
+A Function to fuse calculation for Scale, Zero Point and FP32->8Bit convertion of FP32 Input data.
+Outputs Scale, ZeroPoint and Quantized Input for a given FP32 Input)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    FusedQuantizeLinear,
```

Comment:
Minor nit: The "Fused" part is more of an implementation-detail or concern. It would be better to give it a name that describes what the function is intended to do. But I can see naming it is a bit challenging since the name "QuantizeLinear" is already used up. Would "DynamicLinearQuantization" or some name like that work?

Reply:
Agree... I was struggling with the right name for this so chose to go with Fused... I will change it to DYamicLinearQuantization